src/d3-ext/profile.js

```diff
@@ -58,13 +62,13 @@ ngeo.profile = function() {
    * @type {function(Object)}
    * Hover callback function.
    */
-  var hoverCallback = goog.nullFunction;
+  var hoverCallback = options.hoverCallback || goog.nullFunction;
```

Comment:
Using `goog.isDef` would be less readable, is it really an improvement?
`var hoverCallback = options.hoverCallback || goog.nullFunction;`
> var hoverCallback = goog.isDef(options.hoverCallback) ?  options.hoverCallback :
>   || goog.nullFunction;

Wouldn't it make sense to allow the short syntax when dealing with objects?